src/executeCommandLine/executeCommandLine.ts

```diff
@@ -1014,6 +1015,7 @@ function createWatchOfConfigFile(
     updateWatchCompilationHost(system, cb, watchCompilerHost);
     watchCompilerHost.configFileParsingResult = configParseResult;
     watchCompilerHost.extendedConfigCache = extendedConfigCache;
+    watchCompilerHost.jsDocParsingMode = jsDocParsingMode;

```

Comment:
This (and one below) were missed in my last PR; I only found this by making the parameter required and bubbling that up to the top until I got everything compiling, then made it all optional again.

Reply:
is this not handled by updateWatchCompilationHost above ?

Reply:
Oh, they are, oops! I'll remove them.